pyproject.toml

```diff
@@ -44,6 +44,7 @@ pyyaml = "*"
 requests = { version = "*", optional = true }
 starlette = "*"
 typing-extensions = "*"
+itsdangerous = "^2.1.2"
```

Comment:
What is this for?

Reply:
Session Middleware dependency.

Reply:
Oh I see, maybe we should just depend on `starlette[full]` for test environment.

Reply:
Even for the user environment, `itsdangerous` is required to use `SessionMiddleware`.

Reply:
I don't see any justification in this PR for adding itsdangerous as a prod dependency for everyone who uses starlite.